release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -208,6 +208,14 @@ If you have already upgraded your cluster to {product-title} {product-version} a
 * An IPv6 network for API and Ingress services on the second `machineNetwork` configuration.
 
 For more information, see xref:../networking/ovn_kubernetes_network_provider/converting-to-dual-stack.html#nw-dual-stack-convert_converting-to-dual-stack[Converting to a dual-stack cluster network] in _Converting to IPv4/IPv6 dual-stack networking_.
+[id="ocp-4-16-networking-metallb-frr-k8s_{context}"]
+==== Integration of MetalLB and FRR-K8s (Technology Preview)
+
+This release introduces `FRR-K8s`, a Kubernetes based `DaemonSet` that exposes a subset of the `FRR` API in a Kubernetes-compliant manner.
```

Comment:
ours docs are *everywhere* with DaemonSet. I see `DaemonSet`, daemon set, Daemonset, daemonset. I do not think it needs to be in backticks, but I think that properly it is DaemonSet. But I'll let you decide how to move forward with it.

This is one of those terms I think needs added to our glossary.

Reply:
While I agree that there should be a glossary update, the doc guidelines do use this object as a specific example:
* https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc?plain=1#L1972
* https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc?plain=1#L2007

Reply:
but not a merge blocker IMO.

Reply:
TIL

Reply:
I only found it by searching the repo for DaemonSet :)